llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -2378,7 +2378,8 @@ void InnerLoopVectorizer::scalarizeInstruction(const Instruction *Instr,
     AC->registerAssumption(II);
 
   // End if-block.
-  bool IfPredicateInstr = RepRecipe->getParent()->getParent()->isReplicator();
+  VPRegionBlock *Parent = RepRecipe->getParent()->getParent();
```

Comment:
(strictly speaking, qualifies as `GrandParent` ;-))

Reply:
Perhaps we should introduce `getParentRegion` or `getRegion`, similar to `Instruction::getFunction`?

Reply:
Maybe `getEnclosingRegion()`, to complement and serve `getEnclosingLoopRegion()`?